routes/index.js

```diff
@@ -39,6 +39,7 @@ exports = module.exports = function (app) {
 	app.get('/products', routes.views.products);
 	app.get('/signin', routes.views.login);
 	app.get('/signup', routes.views.signup);
+	app.get('/login2', routes.views.login2);
 
```

Comment:
as I mentioned on pull request comment, please test login2 and replace with login